cuda/test/solver/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-ginkgo_create_test(lower_trs_kernels ADDITIONAL_INCLUDES ${CUDA_INCLUDE_DIRS})
-ginkgo_create_test(upper_trs_kernels ADDITIONAL_INCLUDES ${CUDA_INCLUDE_DIRS})
+ginkgo_create_test(lower_trs_kernels ADDITIONAL_INCLUDES ${CUDAToolkit_INCLUDE_DIRS})
```

Comment:
maybe just use `ginkgo_create_cuda_test` and just always link with the toolkit.

Reply:
done